turn_by_turn/xtrack_line.py

```diff
@@ -49,18 +70,18 @@ def convert_to_tbt(xline: Line) -> TbtData:
         TypeError: If the input is not a valid ``xtrack.Line``.
         ValueError: If no monitors are found or data is inconsistent.
     """
-    if not HAS_XTRACK:
+    if not xt:
```

Comment:
I would prefer a `try ... except` for the `xtrack` import here The raised message (could be logged too) can be the same.